src/webview/js/js.js

```diff
@@ -662,7 +670,13 @@ const handleLongPress = (target: Element) => {
 
   sendMessage({
     type: 'longPress',
-    target: target.matches('.header') ? 'header' : target.matches('a') ? 'link' : 'message',
+    target:
+      target.matches('.header')
+      || (target.parentNode instanceof Element && target.parentNode.matches('.header'))
+        ? 'header'
+        : target.matches('a')
+          ? 'link'
+          : 'message',
```

Comment:
Hmm, this is getting messy.
* If we do end up with logic looking like this, it should definitely get pulled out into a local variable -- it's too complex to belong inline in a function call like this one.
* How about using `closest` instead? See [docs](https://developer.mozilla.org/en-US/docs/Web/API/Element/closest), and example usage for `.message_inline_image a` and friends above.
  * This could also simplify the other part of the diff, above.

Reply:
I agree, was going to ask about that. Fixed! 😄.